common/source/docs/common-lua-scripts.rst

```diff
@@ -289,6 +366,35 @@ The relay library proivdes access to controlling relay outputs.
 
 - :code:`toggle(relay_num)` - Toggles the requested relay on or off.
 
+
+Servo_Channels (SRV_Channels:)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- :code:`find_channel(output function)` - Returns first servo output number -1 of an output assigned "output function" (See ``SERVOx_FUNCTION`` parameters ). False if none is assigned.
+
+Servo Output
+~~~~~~~~~~~~
+
+This is special case that is not generated via the binding methods.
+
+- :code:`servo.set_output (function number, PWM value)` -Sets servo outputs of type "function number" to PWM values between 1000 and 2000.
+
+
+Examples
+========
+
+``local current_pos = ahrs:get_position()``  Allocates current_pos as a local variable, uses the AHRS get_position method to assign it the current position as a Location data type if available, or nil if not.
+
+``global groundspeed = gps::get_velocity_NE(gps:primary_sensor())`` Allocates and sets a global scripting variable, groundspeed, to the ground speed of the vehicle as reported by the primary GPS.
```

Comment:
Global isn't a keyword. Why do you have 2 colon's? There is no such thing as get_velocity_NE on a GPS object, did you mean AHRS? This clearly hasn't been tested.

Reply:
not familiar with LUA, figured it has a global declaration...it does, but its by NOT using local to declare...dont have time to become a LUA programming expert...brain fart on function call...
and I dont test code...just try to write docs...I rely on experts to correct since the docs were not provided at time of addition of the features..